extensions/kubernetes/vanilla/deployment/src/main/java/io/quarkus/kubernetes/deployment/ChangeDeploymentTriggerDecorator.java

```diff
@@ -1,16 +1,10 @@
 package io.quarkus.kubernetes.deployment;
 
-import io.dekorate.kubernetes.decorator.Decorator;
 import io.dekorate.openshift.decorator.ApplyDeploymentTriggerDecorator;
 
 public class ChangeDeploymentTriggerDecorator extends ApplyDeploymentTriggerDecorator {
 
     public ChangeDeploymentTriggerDecorator(String containerName, String imageStreamTag) {
         super(containerName, imageStreamTag);
     }
-
-    @Override
-    public Class<? extends Decorator>[] after() {
-        return new Class[] { ApplyDeploymentTriggerDecorator.class, RemoveDeploymentTriggerDecorator.class };
-    }
```

Comment:
Do we need this change?

Reply:
Same goes for change decorators. 
No, things may work even without those changes but it will be circumstantial.

Reply:
Same. I am wrong, let me completely wipe them.